mediathread/main/views.py

```diff
@@ -891,6 +891,7 @@ def create_course(self, form, affil):
 
         # Add the current user as an instructor.
         c.faculty_group.user_set.add(self.request.user)
+        c.add_detail('instructor', self.request.user.profile.fullname)
```

Comment:
The courseaffils library has a [get_public_name](https://github.com/ccnmtl/django_courseaffils/blob/master/courseaffils/lib.py#L64) function that should be preferred here. (Also available as a template tag [`{% public_name for user %}`](https://github.com/ccnmtl/django_courseaffils/blob/master/courseaffils/templatetags/coursetags.py#L119). The anonymize functionality is handled in that layer.